packages/atomic/src/components/facets/atomic-segmented-facet-scrollable/atomic-segmented-facet-scrollable.tsx

```diff
@@ -88,7 +88,7 @@ export class AtomicSegmentedFacetScrollable implements InitializableComponent {
       return;
     }
 
-    const pixelsToScroll = container.clientWidth * 0.75;
+    const pixelsToScroll = (container.clientWidth - 100) * 0.75;
```

Comment:
How about getting the refs from the buttons and doing something like `const pixelsToScroll = (container.clientWidth - this.arrowRef.clientWidth * 2) * 0.75;`